src/pages/Room/index.js

```diff
@@ -168,10 +156,7 @@ const Room = () => {
             {loading ? <Loading /> : null}
             <ErrorMessage>{error ? error : null}</ErrorMessage>
             <PayloCheckout onClick={handleCheckout}>
-              <img
-                src={data.cashback_button_image || checkoutButton}
-                alt="checkout button"
-              />
+              <img src={buttonImage || checkoutButton} alt="checkout button" />
```

Comment:
@KanRMUTL I look at code. checkout button is url of loading image. could you change the name to related to loading ?

I think we can create another component called `CheckoutButton` and group PayloCheckout and img to this component

Reply:
fixed

Reply:
I created the `CheckoutButton` component